### DISCUSSION TITLE
About blog2 plugin

### DISCUSSION THREAD

**Discussion by NeserCode** [vuepress-theme-hope/vuepress-theme-hope#2367] [Q&A] [answered]
I use the vuepress v2 default theme with blog2 plugin. Problem is the getInfo function cannot get the git plugin's data, but i need data in git plugin. 
So, there have some solutions about this?
I need help.

**Comment by Mister-Hope** [comment, marked as answer]
You should make sure git plugin is called   before blog plugin.

I would recommend you disable git plugin in default theme, then manually called it before blog plugin.
